### ISSUE TITLE
Got charged for the free http load balancer 

### ISSUE THREAD

**Issue by sjbr** [murphye/cheap-gke-cluster#25]
"Use a Regional (rather than Global) HTTP Load Balancer which is currently free as a Beta preview"

That isn't free and adds around $20 monthly.

**Comment by murphye** [comment]
Yes, regional LBs are no longer Beta (and free). They were earlier this year.
